debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java

```diff
@@ -309,6 +309,10 @@ private void updateEventsStats(Connection conn, Map<Pair<String, String>, Map<St
         for (var entry : eventCountDeltaPerTable.entrySet()) {
             Statement tableWiseStatsUpdateStmt = conn.createStatement();
             Pair<String, String> tableQualifiedName = entry.getKey();
+            String schemaName = "";
+            if ((sourceType.equals("postgresql")) && (!tableQualifiedName.getLeft().equals("public"))){
```

Comment:
Alternatively, for Oracle and MySQL, set schemaName to empty string. No special handling needed for PG.
On the voyager side, treat empty schemaName as `tconf.TargetSchema`.

Reply:
For Oracle/Mysql, if i set schemaName to empty, and on voyager side treat empty SchemaName as `tconf.TargetSchema`, it won't match, right?

We sort of follow this logic in a lot of places (to determine snapshot file names/schema file names/etc. The only case where we really need to fully qualify the table name as `<schemaName>.<tableName>` is PG source + non-public schema. I all other cases, the schema name is implicit (i.e. it's the same as `conf.TargetSchema` essentially)

Now, we have to set schemaName as empty in all default-cases so that we can combine the stats of a table exported from source(export data) as well as from target (ff syncrhonize). - https://github.com/yugabyte/yb-voyager/pull/1038 . 

Let me know if there is a better/cleaner way of handling this!

Reply:
If i understood the above discussion correctly, then what Amit is saying is almost similar(just if condition will be different) as what already been done here.
Because in case of postgresql, we don't allow the `target-schema` flag at import side and whatever schema names are on source are followed on target side also.

So yeah just `empty` string for Oracle and MySQL will be okay, and seems better choice.